pkg/util/log/log.go

```diff
@@ -56,6 +58,21 @@ func newPrometheusLogger(l logging.Level, format logging.Format, reg prometheus.
 			Name:      "log_messages_total",
 			Help:      "Total number of log messages.",
 		}, []string{"level"}),
+		timeSpent: promauto.With(reg).NewHistogram(prometheus.HistogramOpts{
+			Namespace: "loki",
+			Name:      "log_messages_time_spent",
+			Help:      "Total time blocking while writing logs to output file.",
+			Buckets: []float64{
```

Comment:
could be `prometheus.ExponentialBuckets(100 * 1e9, 10, 8) // 1e9 - nanoseconds`. ?

start with 100 nanoseconds, with factor of 10, upto 8 times. Meaning 100 nanoseconds to 1 second.

Reply:
Sure, but I prefer to be explicit